exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/http/trace/OtlpHttpSpanExporterBuilder.java

```diff
@@ -93,13 +95,25 @@ public OtlpHttpSpanExporterBuilder setTrustedCertificates(byte[] trustedCertific
 
   /**
    * Sets ths client key and the certificate chain to use for verifying client when TLS is enabled.
-   * The key must be PKCS8, and both must be in PEM format.
+   * The key must be PKCS8, and both must be in PEM format. This will replace any other key manager
+   * that has been configured with this method or with setKeyManager().
```

Comment:
`setKeymanager` doesn't exist. I noticed its also referenced on the javadoc for `OtlpHttpSpanExporterBuilder#setClientTls` as well. Should remove references in both places.